docs/tasks/early_adopter_upgrade.md

```diff
@@ -0,0 +1,83 @@
+---
+sidebar_position: 2
+---
+
+# Upgrade Instructions for Early Adopters
+
+If you were part of the early adopter programme and using Rancher Turtles v0.5.x or less then you need to follow these instructions to use the GA version.
+
+:::note
+The early adopter programme was for non-production use. If you where using Turtles in a production, please contact Rancher support before upgrading.
+:::
+
+## Overview
+
+To upgrade to the GA version of Rancher Turtles you will need to do the following:
+
+1. Delete any child CAPI cluster definitions
+2. Helm uninstall Rancher Turtles
+3. Clean-up orphaned resources
+4. Install the GA version
+
+## Steps
+
+:::danger
+These steps are destructive to child clusters created using CAPI. If you have any concerns, contact Rancher support before proceeding.
+:::
+
+1. Delete any CAPI child cluster definitions and wait for CAPI to fully delete the child clusters
+2. Delete the CAPI providers installed:
+
+```bash=
+kubectl delete capiproviders.turtles-capi.cattle.io -n capi-kubeadm-control-plane-system kubeadm-control-plane
+kubectl delete capiproviders.turtles-capi.cattle.io -n capi-kubeadm-bootstrap-system kubeadm-bootstrap
+kubectl delete capiproviders.turtles-capi.cattle.io -n capi-system cluster-api
+kubectl delete ns capi-kubeadm-control-plane-system
+kubectl delete ns capi-kubeadm-bootstrap-system
+kubectl delete ns capi-system
+```
+
+2. Run the following to uninstall the Turtles extension:
+
+```bash=
+helm uninstall -n rancher-turtles-system rancher-turtles --cascade foreground --wait
+kubectl delete ns rancher-turtles-system
+```
+
+3. Run the following to delete any orphaned resources:
+
+```bash=
+kubectl delete deployments.apps/capi-controller-manager -n capi-system --ignore-not-found=true
+kubectl delete deployments.apps/capi-kubeadm-bootstrap-controller-manager -n capi-kubeadm-bootstrap-system --ignore-not-found=true
+kubectl delete deployments.apps/capi-kubeadm-control-plane-controller-manager -n capi-kubeadm-control-plane-system --ignore-not-found=true
+kubectl delete validatingwebhookconfigurations.admissionregistration.k8s.io capi-validating-webhook-configuration capi-kubeadm-bootstrap-validating-webhook-configuration capi-kubeadm-control-plane-validating-webhook-configuration --ignore-not-found=true
+kubectl delete mutatingwebhookconfigurations.admissionregistration.k8s.io capi-mutating-webhook-configuration capi-kubeadm-bootstrap-mutating-webhook-configuration capi-kubeadm-control-plane-mutating-webhook-configuration --ignore-not-found=true
```

Comment:
should we add rke2 deployments/webhooks?

Reply:
resolved by https://github.com/rancher/turtles-docs/pull/79#discussion_r1547873899